static/assets/rpc-methods/notify_interactive_flow_completed.json

```diff
@@ -91,6 +91,40 @@
         ]
       }
     },
+    {
+      "name": "fee_details",
+      "description": "Description of fees charged by the anchor",
```

Comment:
Should we update the description so it's consistent with the ones in the other RPCs?

Reply:
Done